r/tests/testthat/test-read-write.R

```diff
@@ -119,7 +119,7 @@ test_that("reading/writing a raw vector (sparklyr integration)", {
     as.data.frame(RecordBatchStreamReader$create(x)$read_next_batch())
   }
   bytes <- write_to_raw(example_data)
-  expect_is(bytes, "raw")
+  expect_true(is.raw(bytes))
```

Comment:
Aha, we can! I was getting mixed up as I'd been accidentally using `arrow::type` instead of `base::typeof` to inspect objects, everything makes a lot more sense now.